src/Commands/Setup.php

```diff
@@ -88,6 +88,8 @@ private function publishConfig(): void
         $this->setupRoutes();
 
         $this->runMigrations();
+
+        $this->setSecurityCSRF();
```

Comment:
Can you move this line above the `$this->runMigrations()`?
It seems to me that a good order would be to set up the configuration files and then run the migration last.

Reply:
**Done**, and I made some changes to the test.
@kenjis please check again.